Dockerfile

```diff
@@ -7,6 +7,8 @@ ARG LOG4J_VERSION
 ARG JMX_PROMETHEUS_VERSION
 
 ENV OTEL_SERVICE_NAME=geoserver
+ENV RUN_AS_ROOT=true
+ENV OTEL_LOGS_EXPORTER=none
```

Comment:
what are the options ? needs docs

Reply:
both of them are internal options for geoserver and the otel instrumentation, they should not be changed.